src/open_inwoner/accounts/models.py

```diff
@@ -235,13 +235,15 @@ def get_active_contacts(self):
     def get_contacts_for_approval(self):
         return self.contacts_for_approval.filter(is_active=True)
 
-    def get_all_contacts(self):
```

Comment:
I removed this method as it complicated things when we want to refer to different kind of contacts.